src/js/modules/infragistics.util.js

```diff
@@ -5477,6 +5477,19 @@
 		return item1 ? 1 : -1;
 	};
 
+	// Check wheather certain array of values is equal to another array
+	$.ig.util.areArraysEqual = function (array1, array2) {
+		if (!array1 || !array2) { return false; }
+
+		if (array1.length !== array2.length) { return false; }
+
```

Comment:
You can also include a check for where the two arrays are the same reference. Then there is no need to go through all the members and check them value by value.

There is also a fundamental flaw in this function, which is why unit tests need to be written. Given these two arrays:

`[1, 2, 3]` and `[1, 1, 1]` The function will return `true`